lib/forgery/forgery/lorem_ipsum.rb

```diff
@@ -28,7 +28,7 @@ def self.character(options={})
   end
 
   def self.characters(quantity=10, options={})
-    options.merge!(:random_limit => lorem_ipsum_characters.length-quantity) if quantity.is_a?(Fixnum)
+    options.merge!(:random_limit => lorem_ipsum_characters.length-quantity) if quantity.is_a?(0.class)
```

Comment:
The ultimate goal is to set `options[:random_limit]` if given `quantity` is an integer. In that case, can't we just check `quantity.is_a?(Integer)` instead depending on whatever `0.class` returns (which btw looks to me  as a "hacky" solution 😉)? Or there are some other benefits from using your solution instead of `quantity.is_a?(Integer)`?